package_picks/package-pick-8.19~2024.01+beta1.sh

```diff
@@ -119,23 +119,31 @@ then
   fi 
 
   # Code extraction
-  PACKAGES="${PACKAGES} coq-simple-io.1.8.0"
+  PACKAGES="${PACKAGES} coq-simple-io.1.9.0"
 
   # Proof automation / generation / helpers
   PACKAGES="${PACKAGES} coq-menhirlib.20231231 menhir.20231231"
   PACKAGES="${PACKAGES} coq-equations.1.3+8.19"
   PACKAGES="${PACKAGES} coq-aac-tactics.8.19.0"
   PACKAGES="${PACKAGES} coq-unicoq.1.6+8.19"
+<<<<<<< HEAD
   if [[ "$OSTYPE" != cygwin ]]
   then
     PACKAGES="${PACKAGES} coq-mtac2.1.4+8.19" # build issues on Windows
   fi
+=======
```

Comment:
@rtetley : This file has merge conflicts in it. You need to resolve this before pushing.

Reply:
huh ? weird, I rebased and fixed.. Must have slipped my attention...